src/models/util/ParseCalChart3File.ts

```diff
@@ -0,0 +1,251 @@
+/**
+ * CalChart3 was a bytestream with a series of 4 char codes describing the next section.
+ * In general, the layout is <4char> <size_of_data> <data...> <"END "> <4char>, with
+ * the exception of the first "INGL" block and the version
+ * the Version block is either "GURK", for CalChart3.0, or "GUXY" where X and Y are version 
+ * numbers in hex.
+ * 
+ * For CalChart 3.3 and later there are consistent 4char blocks that contain more elements
+ * of the show.
+ */
+
+import Show from '../Show';
+import StuntSheet from '../StuntSheet';
+import StuntSheetDot from '../StuntSheetDot';
+
+/**
+ * LoadShow
+ * 
+ * @param buffer The Byte array of data to load.
+ * @returns Returns either a new [Show] or null if the Byte array is ill-formed.
+ */
+export default function LoadShow(buffer : ArrayBuffer) {
```

Comment:
In general, functions should be camel case and use ES6 arrow syntax
```suggestion
export default const loadShow = (buffer: ArrayBuffer): Show | null => {
```

Reply:
Done in next pr